test/types/mongodb.test-d.ts

```diff
@@ -15,6 +16,8 @@ expectDeprecated(Collection.prototype.count);
 expectDeprecated(AggregationCursor.prototype.geoNear);
 expectDeprecated(Topology.prototype.unref);
 expectDeprecated(Db.prototype.unref);
+// expectDeprecated(ObjectID); // not sure why tsd can't assert the deprecation
```

Comment:
I think this is because it's root level. I tried this out and it worked:

```
import * as AllExports from '../../src';
expectDeprecated(AllExports.ObjectID);
expectNotDeprecated(AllExports.ObjectId);
```